src/main/java/seedu/duke/command/CommandType.java

```diff
@@ -5,14 +5,16 @@
  *  <p></p>
  *  {@link #LIST}
  *  {@link #ADD}
+ *  {@link #ADDTORECIPE}
  *  {@link #VIEW}
  *  {@link #DELETE}
  *  {@link #HELP}
```

Comment:
@YatPang DELETEFROMRECIPE is missing a link.
```suggestion
 *  {@link #DELETEFROMRECIPE}
 *  {@link #HELP}
```

Reply:
Mark as resolved